api/controllers/DashboardController.js

```diff
@@ -16,275 +19,400 @@ var dashboardController = {
      * @param res
      *
      */
-    dashboard: function (req, res) {
-        if (!(req.session.passport && req.session.passport.user)){
-            return res.redirect(sails.config.customURLs.userServiceURL + '/sign-in');
+    dashboard(req, res) {
```

Comment:
I've noticed that if casebook is down or returns a 500 error, the dashboard no longer renders. A Page not found error is shown instead.
Can you test this simulating when casebook is down? It should still render the dashboard.

Reply:
Fair point. This is a similar thing or a similar fix to the comment you raised here.
https://github.com/UKForeignOffice/loi-application-service/pull/286#discussion_r721218250

It should be sorted now, please pull the code again and let me know if you have any issues.